core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/CoarseMesosSchedulerBackend.scala

```diff
@@ -63,20 +64,25 @@ private[spark] class CoarseMesosSchedulerBackend(
   // Maximum number of cores to acquire (TODO: we'll need more flexible controls here)
   val maxCores = conf.get("spark.cores.max",  Int.MaxValue.toString).toInt
 
+  val maxExecutorsPerSlave = conf.getInt("spark.mesos.coarse.executors.max", 1)
```

Comment:
can we reuse spark.executor.cores by making it from a YARN-only parameter to YARN&Mesos ?